### PR TITLE
Homework 2.v1

### PR DESCRIPTION
**Pull request by badyornyi**
All tasks for Homework 2 (Main Data and Collection Classes) are finished

### CODE REVIEW

File: main_data_classes.rb [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  puts 'Elements with odd indexes are:'
+  i = 1
+  while i < ary.length
+    puts ary[i]
+    i = i+2
+  end
+end
+
+
+#6. There is an array of integers (ary). Puts the index of the last element where ary[0]<ary[i]<ary[-1]
```
Comment (d-konovalov) [inline main_data_classes.rb:139]: And what if your array will have integers in different order?
Please discover other cases

File: main_data_classes.rb [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    if ary[i].even?
+      ary[i] = ary[i] + ary[0]
+    end
+    i+=1
+  end
+  return ary
+end
+
+
+#8. There is a hash where keys and values are strings. Modify it: all keys should be symbols and all values should be integers. If you couldn't modify some value set it to nil
```
Comment (d-konovalov) [inline main_data_classes.rb:179]: 'Modify' means that you should transform your keys-strings to symbols but not deleting them completely

**Review by d-konovalov — CHANGES_REQUESTED** [review]

File: main_data_classes.rb (hunk 1 above)
Comment (badyornyi) [reply main_data_classes.rb:139]: Requested changes for Task#6 are done

File: main_data_classes.rb (hunk 2 above)
Comment (badyornyi) [reply main_data_classes.rb:179]: Requested changes for Task#8 are done.
But I don't sure about its correctness

File: main_data_classes.rb (hunk 1 above)
Comment (d-konovalov) [reply main_data_classes.rb:139]: what if the array would look like this:
```[3,1,5,7,8,0,2,9]```
I would expect index of element 8 as an answer

File: main_data_classes.rb (hunk 2 above)
Comment (d-konovalov) [reply main_data_classes.rb:179]: if you'll call
``` 'my_string'.to_sym```
you get ``` :my_string``` instead

**Review by d-konovalov — CHANGES_REQUESTED** [review]

**Pull request merged**